config/eye.yml.erb

```diff
@@ -58,7 +56,8 @@ processes:
       user_commands:
         rotate: "kill -USR1 {PID}"
 
-<% if is_first_host %>
+# This should only run on 1 instance.
```

Comment:
I don't think that you can/should indent this at this level. Otherwise, it may not be correct yaml.